apps/server/src/adapters/OscAdapter.ts

```diff
@@ -31,15 +33,15 @@ export class OscServer implements IAdapter {
       }
 
       // get second part (command)
-      if (!path) {
+      if (!action) {
```

Comment:
question: is there a reason for this renaming? to my best knowledge `path` is the name of the element.

If it doesnt do much for us other than being different, I would suggest that we keep it as is, no need for a change

Reply:
I dont really mind, the point is to avoid unnecessary changes. we are making large changes to the app and changing something that works can
- introduce sneaky bugs
- result in merge conflicts

If it doesnt provide any value I would prefer skipping it. But if you think this is good and we can say it is harmless (no bugs and no merge conflicts), go ahead